nikola/conf.py.in

```diff
@@ -562,6 +562,29 @@ GITHUB_COMMIT_SOURCE = True
 #    ".jpg": ["jpegoptim --strip-all -m75 -v %s"],
 # }
 
+# The executable for the filter "yui_compressor" from nikola.filters can
```

Comment:
I’d prefer something else (matches how we describe other settings):

    # Executable for the "yui_compressor" filter (defaults to 'yui-compressor').

And perhaps name those settings `*_EXECUTABLE`. With YUI Compressor, we should be prepared to take this:

    java -jar ~/yui-compressor.jar